debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/JdbcChangeEventSink.java

```diff
@@ -88,14 +102,20 @@ public void execute(Collection<SinkRecord> records) {
                 continue;
             }
 
+            final CollectionId collectionId = optionalCollectionId.get();
+
             if (record.isTombstone()) {
-                // Skip only Debezium Envelope tombstone not the one produced by ExtractNewRecordState SMT
-                LOGGER.debug("Skipping tombstone record {}", record);
+                LOGGER.debug("processing tombstone record {}", record);
+
+                if (!config.isDeleteEnabled()) {
```

Comment:
@jpechane @Naros I am currently hitting the question why deletes are disabled by default?